todo_app/__init__.py

```diff
@@ -0,0 +1,18 @@
+# Set env variables here 
+import requests
+import os
+from dotenv import load_dotenv
+
+host = 'https://api.trello.com'
+api_key = os.getenv('API_KEY')
```

Comment:
I can't see any documentation that advises the user they'll need to set these environment variables - we should mention them (as well as the setup for Trello required) in your Readme, and also update the .env.template file with the relevant keys